cli/src/stake.rs

```diff
@@ -196,6 +196,12 @@ impl StakeSubCommands for App<'_, '_> {
                         .validator(is_valid_signer)
                         .help("Source account of funds [default: cli config keypair]"),
                 )
+                .arg(
+                    Arg::with_name("force")
+                        .long("force")
+                        .takes_value(false)
+                        .help("Don't enforce the minimum delegation check")
+                )
```

Comment:
I didn't make it hidden because this is a sanity check and users may actually want to force creation of such a stake account, as it doesn't actually fail with a ProgramError, in this context do you still think it should be hidden?